app/views/enrollment_mailer/aidants_connect/validate_application.text.erb

```diff
@@ -1,5 +1,10 @@
+Bonjour,
+
 Votre demande d’habilitation a été validée. Les aidants professionnels au sein de votre structure seront contactés très prochainement par courriel afin de suivre une formation à la prise en main d’Aidants Connect. Retrouvez dès maintenant tous les détails du processus d’habilitation sur notre page dédiée : https://aidantsconnect.beta.gouv.fr/habilitation .
 <% if @message %>
```

Comment:
I'd like to remove this feature totaly. 

Can we remove the "partial edition" feature
 ?

Reply:
Like I said in [this comment](https://github.com/betagouv/signup-back/pull/200#issuecomment-838692594) I can do that.

Reply:
a59a933